data/network_security

```diff
@@ -0,0 +1,4 @@
+fofa.so
+t00ls.cc
+t00ls.net
+tools.team
```

Comment:
BTW, can you provide the accessible sites' URLs under this domain?

Reply:
1. www.fofa.so
2. www.t00ls.cc
3. www.t00ls.net

Both are official websites

Reply:
> * [www.fofa.so](http://www.fofa.so)
> * [www.t00ls.cc](http://www.t00ls.cc)
> * [www.t00ls.net](http://www.t00ls.net)
> 
> Both are official websites

I mean `*.tools.team`

Reply:
- About fofa.so

`classic.fofa.so`

`faweb.fofa.so`

- About `t00ls.cc` `t00ls.net` `tools.team` 

These three domains are all the same official site, just with different domain names

These 3 domains do not have subdomains, only www domains

Reply:
![image](https://user-images.githubusercontent.com/32576256/148083522-6d9b5313-b324-4043-9e66-64d2c1778caf.png)

Reply:
- Situation description

No need to add www, you can directly T00ls.team for access.

The domain name will be redirected to t00ls.cc after 302 hops

Also attached is the official announcement

![image](https://user-images.githubusercontent.com/29684167/148156644-b6c43854-be11-48fd-85a0-e5efbc86b282.png)